docs/investigation/configuration/2022-06-14-policies.md

```diff
@@ -192,7 +192,7 @@ policies:
 
 ### Route notifications to a given channel based on the Kubernetes Namespace
 
-Currently, you can send notification to non-default channel using [annotation](https://botkube.io/usage/#send-notification-to-non-default-channel).
+Currently, you can send notification to non-default channel using [annotation](https://docs.botkube.io/usage/#send-notification-to-non-default-channel).
```

Comment:
I cannot find this section anymore 🤔 

The last mention it is in 0.14: https://docs.botkube.io/0.14/usage/#send-notification-to-non-default-channel